### PR TITLE
[Documentation] Camel case all keys in api docs

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

Continuation of #7556

### CODE REVIEW

File: docs/api/zones.rst [hunk 1]
```diff
@@ -237,7 +237,7 @@ Definition
 | limit                                 | query          | *(optional)* Number of items to display per page, |
 |                                       |                | by default = 10                                   |
 +---------------------------------------+----------------+---------------------------------------------------+
-| sorting['name_of_field']['direction'] | query          | *(optional)* Field and direction of sorting,      |
+| sorting['nameOfField']['direction']   g| query          | *(optional)* Field and direction of sorting,      |
```
Comment (tuka217) [inline docs/api/zones.rst]: remove `g`

File: docs/api/customers.rst [hunk 2]
```diff
@@ -87,7 +87,7 @@ Example
 
 .. code-block:: bash
 
-    curl http://sylius.dev/api/v1/customers/ \
+    $ curl http://127.0.0.1:8000/api/v1/customers/ \
```
Comment (tuka217) [inline docs/api/customers.rst]: `demo.sylius.org`?

File: docs/api/customers.rst [hunk 3]
```diff
@@ -620,19 +620,707 @@ Exemplary Response
...
+                            },
+                            "_links":{
+                                "order":{
+                                    "href":"\/api\/v1\/orders\/2"
+                                },
+                                "product":{
+                                    "href":"\/api\/v1\/products\/756b4849-2063-3044-852c-6aa261eddff7"
+                                },
+                                "variant":{
+                                    "href":"\/api\/v1\/products\/756b4849-2063-3044-852c-6aa261eddff7\/variants\/756b4849-2063-3044-852c-6aa261eddff7-variant-0"
```
Comment (tuka217) [inline docs/api/customers.rst]: 🦄  Maybe we should use product and variant with shorter code?

**Comment by pjedrzejewski** [comment]
Thanks Łukasz!

**Pull request merged**